modules/dnn/src/net_impl_fuse.cpp

```diff
@@ -737,7 +737,7 @@ void Net::Impl::fuseLayers(const std::vector<LayerPin>& blobsToKeep_)
                           inp_i_data->layerInstance->type != "Permute" &&
                           inp_i_data->layerInstance->type != "Reorg" &&
                           inp_i_data->layerInstance->type != "Eltwise" &&
-                          inp_i_data->layerInstance->type != "NaryEltwise" &&
+                        //   inp_i_data->layerInstance->type != "NaryEltwise" && // link to the issue: https://github.com/opencv/opencv/issues/24721
```

Comment:
@dkurt I guess they want your comment on this change.

In my opinion, we do not need to comment out this line as https://github.com/opencv/opencv/pull/24834 has mostly resolved the issue. Previously it fuses `NaryEltwise` and `Concat` regardless the CPU fallback in `NaryEltwise`. So there is data both in the host and device in the output of `Concat`, leading to the missing data reported in those related issues. The CPU fallback happened previously when broadcasting or operation is not supported. The broadcast support is done, leaving the risk of unsupported operation. However, it should also be fine since common operations hves been supported (add, sub, mul, div) and we do not get performance regression with disabling this fusion.

Reply:
@fengyuentau, yes, I understand this but this PR before https://github.com/opencv/opencv/pull/24834 merged shows that even with disabled broadcast all the tests passed so having an exclusion like this is redundant.

Reply:
> shows that even with disabled broadcast all the tests passed

All tests were passed because:

- there is no `naryeltwise + concat` model case in the tests before this PR,
- this PR added the test case but commented out the `naryeltwise + concat` fusion.